functions/checkMatch.js

```diff
@@ -0,0 +1,43 @@
+/** check if two cards match **/
+/* This function:
+*       1. receives an array of all of the card's locations
+*       2. receives an array of chosen card locations
+*       3. uses the values in the chosen card array to determine which locations to check in the cards array
+*       4. gets the values stored in those locations in the card array
+*       5. checks if the two values in the card array match
+*       6. returns true if both match
+*       7. returns false if they do not match
```

Comment:
I'm not sure we need a summary here AND line-by-line comments throughout the function as well?